src/components/navbar.jsx

```diff
@@ -2,40 +2,38 @@ import { useEffect } from "react";
 import { useState } from "react";
 import Logo from "../assets/logo.png";
 
-
 const Navbar = () => {
-  const [theme, setTheme] = useState("dark");
+  const defaultTheme = localStorage.getItem("theme") || "light";
+  const [theme, setTheme] = useState(defaultTheme);
   const element = document.documentElement;
-  
+
   const icons = [
     {
       icon: "moon",
-      text: "light"
+      text: "light",
     },
     {
       icon: "sunny",
-      text: "dark"
-    }
+      text: "dark",
```

Comment:
I believe, there is another here. Am I the one missing something perhaps?
```suggestion
      text: "dark"
```

Reply:
Oh I see it's because my code formatter. eslint has format the code automatically